test/integration/controller.v2/trainingruntime_controller_test.go

```diff
@@ -0,0 +1,128 @@
+/*
```

Comment:
@tenzen-y @akshaychitneni What do you think about moving the CEL validation and defaults integration tests to [the `webhook.v2` directory](https://github.com/kubeflow/training-operator/tree/master/test/integration/webhook.v2)?

I noticed that we do it for:
- Kueue: https://github.com/kubernetes-sigs/kueue/blob/main/test/integration/webhook/core/clusterqueue_test.go#L69
- JobSet: https://github.com/kubernetes-sigs/jobset/blob/main/test/integration/webhook/jobset_webhook_test.go#L40

I think, it makes more sense given that we don't have controller for TrainingRuntime and ClusterTrainingRuntime currently.
Additionally, CEL is part of Kubernetes admission control process.

Reply:
Makes sense.